pkg/factories/query_builder_factory.go

```diff
@@ -0,0 +1,48 @@
+package factories
```

Comment:
As we discussed, this is unfortunately more related to Faktory being named something so ambiguous, there was an intentional choice to name this package "factories" rather than "factory". However, in keeping with Go idiomatic package naming, we have renamed this package to be more specific to query_builder. This is therefore incidentally resolved.